### PR TITLE
LTR 9 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/flowering_of_the_white_tree.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Name:Flowering of the White Tree
+ManaCost:W W
+Types:Legendary Enchantment
+S:Mode$ Continuous | Affected$ Creature.Legendary+YouCtrl | AddPower$ 2 | AddToughness$ 1 | AddKeyword$ Ward:1  | Description$ Legendary creatures you control get +2/+1 and have ward {1}.
+S:Mode$ Continuous | Affected$ Creature.nonLegendary+YouCtrl | AddPower$ 1 | AddToughness$ 1 | Description$ Nonlegendary creatures you control get +1/+1.
+SVar:PlayMain1:TRUE
+DeckHints:Type$Legendary
+Oracle:Legendary creatures you control get +2/+1 and have ward {1}./nNonlegendary creatures you control get +1/+1.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/flowering_of_the_white_tree.txt:8]: \n

File: forge-gui/res/cardsfolder/upcoming/king_of_the_oathbreakers.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Name:King of the Oathbreakers
+ManaCost:2 W B
+Types:Legendary Creature Spirit Noble
+PT:3/3
+K:Flying
+T:Mode$ BecomesTarget | ValidTarget$ Card.Self,Spirit.YouCtrl+Other | TriggerZones$ Battlefield | Execute$ TrigPhaseOut| TriggerDescription$ Whenever CARDNAME or another Spirit you control becomes the target of a spell, it phases out. (Treat it and anything attached to it as though they don't exist until your next turn.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/king_of_the_oathbreakers.txt:6]: Spell restriction not handled

File: forge-gui/res/cardsfolder/upcoming/long_list_of_the_ents.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Name:Long List of the Ents
+ManaCost:G
+Types:Enchantment Saga
+K:Saga:6:DBNoteType,DBNoteType,DBNoteType,DBNoteType,DBNoteType,DBNoteType,
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/long_list_of_the_ents.txt:4]: extra ,

File: forge-gui/res/cardsfolder/upcoming/long_list_of_the_ents.txt [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+SVar:DBNoteType:DB$ ChooseType | Type$ Creature | Note$ True | SpellDescription$ Note a creature type that hasn't been noted for CARDNAME. When you cast your next creature spell of that type this turn, that creature enters the battlefield with an additional +1/+1 counter on it.
+T:Mode$ SpellCast | ValidCard$ Creature.NotedType | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Secondary$ True | Execute$ TrigEffect | TriggerDescription$ When you cast your next creature spell of that type this turn, that creature enters the battlefield with an additional +1/+1 counter on it.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/long_list_of_the_ents.txt:6]: delayed trigger

File: forge-gui/res/cardsfolder/upcoming/meriadoc_brandybuck.txt [hunk 5]
```diff
@@ -0,0 +1,9 @@
+Name:Meriadoc Brandybuck
+ManaCost:1 G
+Types:Legendary Creature Halfling Citizen
+PT:2/2
+T:Mode$ AttackersDeclaredOneTarget | ValidAttackers$ Halfling.YouCtrl | AttackedTarget$ Opponent | Execute$ TrigFood| TriggerZones$ Battlefield | TriggerDescription$ Whenever one or more Halflings you control attack a player, create a Food token. (It's an artifact with "{2}, {T}, Sacrifice this artifact: You gain 3 life.")
+SVar:TrigFood:DB$ Token | TokenAmount$ 1 | TokenScript$ c_a_food_sac | TokenOwner$ You
+DeckHas:Ability$Tokens|LifeGain & Type$Food|Artifact
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/meriadoc_brandybuck.txt:7]: invalid Ability

File: forge-gui/res/cardsfolder/upcoming/merry_esquire_of_rohan.txt [hunk 6]
```diff
@@ -0,0 +1,11 @@
+Name:Merry, Esquire of Rohan
+ManaCost:R W
+Types:Legendary Creature Halfling Knight
+PT:2/2
+K:Haste
+S:Mode$ Continuous | Affected$ Card.Self+equipped | AddKeyword$ First Strike | Description$ CARDNAME has first strike as long as it's equipped.
+T:Mode$ AttackersDeclared | ValidAttackers$ Card.Self | IsPresent$ Creature.attacking+Other+YouCtrl+Legendary | TriggerZones$ Battlefield | Execute$ TrigDraw | TriggerDescription$ Whenever you attack with NICKNAME and another legendary creature, draw a card.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/merry_esquire_of_rohan.txt:7]: needs NoResolvingCheck

File: forge-gui/res/cardsfolder/upcoming/morgul_knife_wound.txt [hunk 7]
```diff
@@ -0,0 +1,9 @@
+Name:Morgul-Knife Wound
+ManaCost:1 B
+Types:Enchantment Aura
+K:Enchant creature
+A:SP$ Attach | ValidTgts$ Creature | AILogic$ Curse
+S:Mode$ Continuous | Affected$ Creature.EnchantedBy | AddPower$ -3 | AddTrigger$ TrigExile | Description$ Enchanted creature gets -3/-0
+SVar:TrigExile:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ Player.EnchantedController | TriggerZones$ Battlefield | Execute$ TrigExileEnchanted | TriggerDescription$ Enchanted creature gets -3/-0 and has "At the beginning of your upkeep, exile this creature unless you pay 2 life."
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/morgul_knife_wound.txt:7]: Are you sure that finds the right player?

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/morgul_knife_wound.txt:7]: I think so yeah 
image

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/morgul_knife_wound.txt]: you're right it trigger on every player's upkeep. Fixed

File: forge-gui/res/cardsfolder/upcoming/long_list_of_the_ents.txt (hunk 4 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/long_list_of_the_ents.txt]: your problem is all chapters will trigger for every noted type

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/long_list_of_the_ents.txt]: Will this need engine changes or are there similar cards I should look at ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/long_list_of_the_ents.txt]: You might need to convert it into an Effect so you'll have another host you can transfer the type directly

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/long_list_of_the_ents.txt]: hmm you mean putting the choosetype into an effect ? how will it remember the "hasn't been noted" part ?

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/long_list_of_the_ents.txt]: I'll skip this one for now

File: forge-gui/res/cardsfolder/upcoming/king_of_the_oathbreakers.txt [hunk 8]
```diff
@@ -0,0 +1,12 @@
+Name:King of the Oathbreakers
+ManaCost:2 W B
+Types:Legendary Creature Spirit Noble
+PT:3/3
+K:Flying
+T:Mode$ BecomesTarget | ValidTarget$ Card.Self,Spirit.YouCtrl+Other | ValidSource$ Spell | TriggerZones$ Battlefield | Execute$ TrigPhaseOut| TriggerDescription$ Whenever CARDNAME or another Spirit you control becomes the target of a spell, it phases out. (Treat it and anything attached to it as though they don't exist until your next turn.)
+SVar:TrigPhaseOut:DB$ Phases | Defined$ TriggeredTarget
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/king_of_the_oathbreakers.txt:7]: LKICopy

File: forge-gui/res/cardsfolder/upcoming/morgul_knife_wound.txt [hunk 9]
```diff
@@ -0,0 +1,9 @@
+Name:Morgul-Knife Wound
+ManaCost:1 B
+Types:Enchantment Aura
+K:Enchant creature
+A:SP$ Attach | ValidTgts$ Creature | AILogic$ Curse
+S:Mode$ Continuous | Affected$ Creature.EnchantedBy | AddPower$ -3 | AddTrigger$ TrigExile | Description$ Enchanted creature gets -3/-0
+SVar:TrigExile:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You  | TriggerZones$ Battlefield | Execute$ TrigExileEnchanted | TriggerDescription$ Enchanted creature gets -3/-0 and has "At the beginning of your upkeep, exile this creature unless you pay 2 life."
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/morgul_knife_wound.txt:7]: Wrong Description

File: forge-gui/res/cardsfolder/upcoming/morgul_knife_wound.txt [hunk 10]
```diff
@@ -0,0 +1,9 @@
...
+SVar:TrigExileEnchanted:DB$ ChangeZone | Defined$ Enchanted | Origin$ Battlefield | Destination$ Exile | UnlessCost$ PayLife<2> | UnlessPayer$ TriggeredPlayer | SpellDescription$ Exile this creature unless you pay 2 life
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/morgul_knife_wound.txt:8]: Defined seems wrong and Description probably unused?

**Review by tool4ever — APPROVED** [review]

**Pull request merged**